docs/design/metrics.md

```diff
@@ -61,44 +63,13 @@ How the metrics are exposed to the end-user is determined by the CLI.
 
 [^1]: The top-level namespace is added by the CLI using [`metrics_util::layers::PrefixLayer`][metrics_util.PrefixLayer].
 
-### Current metrics
-
-This list may be non-exhaustive.
-
-#### Stage: Headers
-
-- `stages.headers.counter`: Number of headers successfully retrieved
-- `stages.headers.timeout_error`: Number of timeout errors while requesting headers
-- `stages.headers.validation_errors`: Number of validation errors while requesting headers
-- `stages.headers.unexpected_errors`: Number of unexpected errors while requesting headers
-- `stages.headers.request_time`: Elapsed time of successful header requests
-
-#### Component: Transaction Pool
-
-- `transaction_pool.inserted_transactions`: Number of transactions inserted in the pool
-- `transaction_pool.invalid_transactions`: Number of invalid transactions
-- `transaction_pool.removed_transactions`: Number of removed transactions from the pool
-- `transaction_pool.pending_pool_transactions`: Number of transactions in the pending sub-pool
-- `transaction_pool.pending_pool_size_bytes`: Total amount of memory used by the transactions in the pending sub-pool in bytes
-- `transaction_pool.basefee_pool_transactions`: Number of transactions in the basefee sub-pool
-- `transaction_pool.basefee_pool_size_bytes`: Total amount of memory used by the transactions in the basefee sub-pool in bytes
-- `transaction_pool.queued_pool_transactions`: Number of transactions in the queued sub-pool
-- `transaction_pool.queued_pool_size_bytes`: Total amount of memory used by the transactions in the queued sub-pool in bytes
-- `transaction_pool.blob_pool_transactions`: Number of transactions in the blob sub-pool
-- `transaction_pool.blob_pool_size_bytes`: Total amount of memory used by the transactions in the blob sub-pool in bytes
-
-#### Component: Network
-
-- `network.connected_peers`: Number of currently connected peers
-- `network.tracked_peers`: Number of peers known to the node
-- `network.pending_session_failures`: Cumulative number of failures of pending sessions
-- `network.closed_sessions`: Total number of sessions closed
-- `network.incoming_connections`: Number of active incoming connections
-- `network.outgoing_connections`: Number of active outgoing connections
-- `network.total_incoming_connections`: Total number of incoming connections handled
-- `network.total_outgoing_connections`: Total number of outgoing connections established
-- `network.invalid_messages_received`: Number of invalid/malformed messages received from peers
-- `network.propagated_transactions`: Total number of propagated transactions
+### Example metrics
+- [Transcation Pool Metrics](../../crates/transaction-pool/src/metrics.rs)
```

Comment:
hey hi @jenpaff - when trying to run the lint locally (as described here: [`make pr` in CONTRIBUTING.md](https://github.com/paradigmxyz/reth/blob/main/CONTRIBUTING.md?plain=1#L101)) I got this lint error:

<img width="669" alt="image" src="https://github.com/user-attachments/assets/06ce44b6-4220-4cab-a6e3-4b88c6b0a991" />

(There is indeed a typo but it looks like this `make pr` command is not enforced in the CI so it might be outdated docs in the `CONTRIBUTING.md` - not a big deal but still flagging here as new contributors might go over similar problems 🫡)

Reply:
thanks for flagging, will try and fix !